cmd/inspect.go

```diff
@@ -271,44 +292,44 @@ func launchPlugins(config *tflint.Config, fix bool) (*plugin.Plugin, error) {
 		if err != nil {
 			if st, ok := status.FromError(err); ok && st.Code() == codes.Unimplemented {
 				// VersionConstraints endpoint is available in tflint-plugin-sdk v0.14+.
-				return rulesetPlugin, fmt.Errorf(`Plugin "%s" SDK version is incompatible. Compatible versions: %s`, name, plugin.SDKVersionConstraints)
+				return rulesetPlugin, sdkVersions, fmt.Errorf(`Plugin "%s" SDK version is incompatible. Compatible versions: %s`, name, plugin.SDKVersionConstraints)
```

Comment:
This shouldn't be propegated up anymore now that validate is internal to launchPlugins